### PR TITLE
[9.0][FIX] report_xls: Move description to first line

### PR DESCRIPTION
**Pull request by lasley**
I'm not sure why the [RunBot builds are failing](https://runbot.odoo-community.org/runbot/build/3152292) on a non-installable module, but they are and it's killing my OCD. 

This is minimalistic change that ~~hopefully~~ fixes the builds.

### CODE REVIEW

File: report_xls/__openerp__.py [hunk 1]
```diff
@@ -26,8 +26,7 @@
     'author': "Noviat,Odoo Community Association (OCA)",
     'website': 'http://www.noviat.com',
     'category': 'Reporting',
-    'description': """
-Excel report engine
+    'description': """Excel report engine
```
Comment (elicoidal) [inline report_xls/__openerp__.py]: Simply remove the description key. This is not mandatory anymore with the README file.
(even more this becomes redundant and difficult to keep in sync)

Comment (lasley) [inline report_xls/__openerp__.py]: In this instance there isn't a ReadMe, and I was trying to keep the change-set minimal as this isn't a full conversion & I am just trying to fix the build (all 9.0 PRs are failing right now). If this is a block however, I will move this over to a ReadMe using OCA template.

Comment (pedrobaeza) [inline report_xls/__openerp__.py]: You can cherry-commit https://github.com/OCA/reporting-engine/commit/fddca415b8d255b9d877ea92217ea086205ce077 from 8.0

Comment (lasley) [inline report_xls/__openerp__.py]: Excellent call, thanks @pedrobaeza

**Comment by lasley** [comment]
Oops forgot to drop a note saying comments handled after handling comments. Comments handled! 😄

File: report_xls/__openerp__.py (hunk 1 above)
Comment (atchuthan) [reply report_xls/__openerp__.py]: you could remove the description field as README.rst was added

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Although the module is not yet merged, but we can fast-track it anyway.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**